website/docs/reference/dbt-jinja-functions/project_name.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "project_name"
 id: "project_name"
-description: "Use this context variable to return the `name` of root-level project."
+description: "Read this guide to understand the project_name Jinga function in dbt."
```

Comment:
`Jinga` should be `Jinja`

Reply:
https://deploy-preview-3261--docs-getdbt-com.netlify.app/reference/dbt-jinja-functions/project_name

content="Read this guide to understand the project_name Jinja function in dbt."